### DISCUSSION TITLE
laravel site not run locally

### DISCUSSION THREAD

**Discussion by salimahamad** [OpnForm/OpnForm#466] [General]
when i am run laravel locally show this message
====
message": "The route / could not be found.",
    "exception": "Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException",
    "file": "E:\\xampp\\htdocs\\OpnForm\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\AbstractRouteCollection.php",
    "line": 44,
    "trace": [
![image](https://github.com/JhumanJ/OpnForm/assets/49427727/92cd3a72-e5cd-4fbc-95ff-4dd079a3e613)

**Comment by JhumanJ** [comment]
The error says it all: the API doens't have a route `/`. Please make sure to also run the Nuxt app (front-end client)

**Reply by salimahamad** [reply]
yes front-end client run
